tests/client-dbus/tests/udev/test_predict.py

```diff
@@ -62,11 +62,14 @@ def _call_predict_usage(encrypted, device_sizes, *, fs_specs=None, overprovision
     """
     with subprocess.Popen(
         [_STRATIS_PREDICT_USAGE, "pool"]
-        + ["--device-size=%s" % size for size in device_sizes]
+        + [f"--device-size={size for size in device_sizes}"]
         + (
             []
             if fs_specs is None
-            else [("--filesystem-size=%s" % size.magnitude) for _, size in fs_specs]
+            else [
+                (f"--filesystem-size={size.magnitude}" % size.magnitude)
```

Comment:
This line isn't quite right.

Reply:
Whoops; I forgot to clean that up.  I'll fold that into the `Switch Python programs to literal string interpolation` commit.

Reply:
...And since the line is shorter, it needs to be formatted again.